Packs/BitSight/Integrations/BitSightForSecurityPerformanceManagement/BitSightForSecurityPerformanceManagement.yml

```diff
@@ -4,29 +4,27 @@ commonfields:
 name: BitSight for Security Performance Management
 display: BitSight for Security Performance Management
 category: Utilities
-description: "BitSight for Security Performance Management (SPM) enables CISOs to\
-  \ use an external view of security performance to measure, monitor, manage, and\
-  \ report on their cybersecurity program performance over time, and to facilitate\
-  \ a universal understanding of cyber risk across their organization. This improved\
-  \ understanding enables security leaders to make more informed decisions about their\
-  \ cybersecurity program, including where to focus their limited resources in order\
-  \ to achieve the greatest impact, where to spend money, and how to manage their\
-  \ cyber risk more effectively. \nThe data-driven metrics within BitSight indicate\
-  \ if the cybersecurity program is performing up to the expectations set by internal\
-  \ goals and objectives, industry best practices, regulators, customers, and other\
-  \ internal or external stakeholders. The BitSight Security Rating, the industry’s\
-  \ original cybersecurity rating score, provides a trusted metric that reflects the\
-  \ organization’s cybersecurity program performance over time. By combining the insights\
-  \ gained from BitSight SPM with the BitSight Security Rating, security leaders provide\
-  \ a more complete view of their cybersecurity program performance over time and\
-  \ help to bring about a universal understanding of cyber risk to the Board of Directors\
-  \ and other stakeholders. \nBring BitSight findings event information into your\
-  \ security program and leverage Cortex XSOAR's incident management workflows for\
-  \ automation of managing security incidents. This visibility enables you to pinpoint\
-  \ and control the sources of infections in your company infrastructure, seamlessly\
-  \ going from awareness to rapid remediation. The findings information reveals associated\
-  \ IP addresses, destination ports, and more, to assist your company in connecting\
-  \ the security and IT teams to respond faster and more effectively to threats. "
+description: 'BitSight Integration to get company guid, deatils, findings and to create
+  Incidents. '
+detaileddescription: "### Community Contributed Integration\nSupport and maintenance
```

Comment:
Please except my note regarding the description here, this description is very long and Im afraid it will not look good in the UI - https://xsoar.pan.dev/docs/documentation/documentation_tips#for-integrations
You can add this description to the pack README - https://xsoar.pan.dev/docs/documentation/pack-docs

Reply:
Fixed the code